docs/developers_guide.rst

```diff
@@ -83,7 +83,7 @@ Handling of uncertain oracles
 
 Transductive and inductive active learning
 ------------------------------------------
-* possible with distinct X_cand list independent X, y, E_eval, etc.
+* possible with distinct X_cand list independent X, y, X_eval, etc.
```

Comment:
My main inquiry for now is the existence of X_cand that I do not get. If I can pass X and y to fit, with a specific label for unlabeled samples in y, then why can't I do the same for query? In that case, indexing would be completely transparent.

Reply:
With your argument, we could get rid of X_cand, as we can identify candidates by the specific label in (X,y), correct? The problem, we identified with that idea was: If you have a semi-supervised classifier, we would need to fit that on (X,y). Sometimes you only want to consider a selection of candidates for active learning. As this would not be possible, we decided to go with a separate X_cand. Another argument is that if your classifier is already fitted, it would be sufficient to pass X_cand

Reply:
Another argument is that in the stream case, we normally consider single unlabeled  instances, which can be easily given by X_cand